src/copy.jl

```diff
@@ -93,7 +99,10 @@ function copy_model(model::Model)
     reference_map = ReferenceMap(new_model, index_map)
 
     for (name, value) in object_dictionary(model)
-        new_model.obj_dict[name] = getindex.(reference_map, value)
+        if filter_constraints === nothing || 
+                (value isa ConstraintRef && filter_constraints(value))
```

Comment:
This doesn't catch the case where `value` is a container of ConstraintRefs (please add a test).

Reply:
I don't get how you can generate such a case (even if it's mentioned in the docs: https://github.com/jump-dev/JuMP.jl/blob/1d36b2c3c1bb132dc2e97af3f2b17ef544eecbca/src/JuMP.jl#L932) from user code. I don't find any occurrence of this in the test suite either.

Reply:
`@constraint(model, constr[i = 1:2], x[i] <= y[i])` for example. See https://jump.dev/JuMP.jl/v0.21.5/constraints/#Constraint-containers-1.

Reply:
Thanks for your pointer! I'm working on a patch for this.